pkg/util/welcome.go

```diff
@@ -27,7 +27,7 @@ import (
 
 const (
 	minSupportedVersion = 27
```

Comment:
I actually think we need to use 28 here as well - Thoughts @zroubalik, @JorTurFer?

Reply:
In theory, we should as we use N-2 -> 30, 29, 28

Reply:
Agreed!